lua/terrortown/migrations/2024_02_16_23_10_favorites_migration.lua

```diff
@@ -0,0 +1,100 @@
+--- @ignore
+-- Migration file for shop favorites
+if SERVER then
+    return function() end
+end
+
+local oldDatabaseName = "ttt_bem_fav"
+local newDatabaseName = "ttt2_shop_favorites"
+---
+-- Drops an existing sql table with the same name and creates a new one for shop favorites
+-- Then checks out old favorites table and merges them into the new table as good as possible
+-- @realm client
+return function()
+    Dev(1, "Starting to migrate shop favorites.")
+
+    -- Drop existing table
+    local result = nil
+    if sql.TableExists(newDatabaseName) then
+        Dev(1, "SQL table", newDatabaseName, "already exists. It will be dropped!")
+        result = sql.Query("DROP TABLE " .. sql.SQLStr(newDatabaseName))
+    end
```

Comment:
I think you can safely assume this to not exist

Reply:
This is safer though and we make sure the table has the correct format.

Reply:
> I think you can safely assume this to not exist

That would be a fair assumption to make, but given that we can handle this failure case we should do so.